doc/tools/docgram/common.edit_mlg

```diff
@@ -1351,6 +1352,22 @@ module_expr: [
 | DELETE module_expr module_expr_atom
 ]
 
+locatable: [
+| "Ltac" reference
+]
+
+ne_in_or_out_modules: [
+| REPLACE "inside" LIST1 global
+| WITH [ "inside" | "outside" ] LIST1 global
+| DELETE "outside" LIST1 global
+]
+
+searchabout_queries: [
+| DELETE ne_in_or_out_modules
+| REPLACE searchabout_query searchabout_queries
+| WITH [ ne_in_or_out_modules | LIST1 searchabout_query ]
+]
```

Comment:
This edit is incorrect. `searchabout_queries` is in fact `LIST0 [ searchabout_query ] ne_in_or_out_modules`.

Reply:
Right, though I do it a bit differently to end up with `LIST1 searchabout_query ...` here. `searchabout_query` is later renamed to `search_item`.  (Chained edits are hard to follow.)

```
| "Search" searchabout_query searchabout_queries "."
```